src/CommandingOfficers/Ave.java

```diff
@@ -655,14 +653,10 @@ private static class OblidoAbility extends CommanderAbility
       Ave = commander;
       damageMod = new UnitDamageModifier(OBLIDO_BUFF);
       AIFlags = PHASE_TURN_START | PHASE_TURN_END;
-    }
-
-    @Override
-    protected void adjustCost()
-    {
       // One of the big benefits of this power is deforestation, since trees get in Ave's way.
       // Trees are only removed once, so we'll increase cost more slowly to counteract the decreased utility.
-      myPowerCost += 0.5 * CHARGERATIO_FUNDS;
+      costBasis.starRatioPerCast = 555; // 8 stars * 555 = +4440 per cast, vs the old 9000/2=4500 per cast
```

Comment:
Obvious question - why make it different-but-almost-the-same?

Reply:
4500/8 = 562.5, which doesn't seem representable in the integers.
I guess I could have done something like what Nix does - half the star ratio, double the star count, and increase stars by 1 per cast.
I figured you'd have an opinion either way, and it's your CO, and you're reviewing the code. I just picked something so I could move on.
(and I changed everyone else to have linked ability costs, so you may or may not want to adjust her stuff too)

Reply:
The balance here was never exactly fine tuned, so I'm not going to lose sleep over it.